ci/release-image/entrypoint.sh

```diff
@@ -18,4 +18,13 @@ if [ "${DOCKER_USER-}" ]; then
   fi
 fi
 
+# Allow users to have scripts run on container startup to prepare workspace.
+# https://github.com/coder/code-server/issues/5177
+chmod u+x ${ENTRYPOINTD}/*.sh
+sudo chown -R ${USER} ${ENTRYPOINTD}/*
+for f in ${ENTRYPOINTD}/*.sh; do
+  echo "Running Entrypoint: ${f}"
+  bash "${f}"
```

Comment:
Any thoughts on if we could run the script directly instead of through bash?  That way users can use whatever interpreter they want (sh, zsh, or even something entirely different) via the shebang.  Then we could do `${ENTRYPOINTD}/*` as well.

Also I think if any of these scripts exit with non-zero it will cause the entrypoint to exit (due to the `set -e`) so we might want to catch them so they do not prevent code-server from starting (`if ! "${f}" ; then echo "${f} exited with non-zero"; fi`) or something like that.

But then again maybe we do want these to cause a hard exit?  I could go either way personally.  Any thoughts?  @jsjoeio as well.

Reply:
That might not be the worst idea, then people could also include say binary to execute as well.
This would require that the end user does ensure that the `x` bit is set.

My personal preference would be to hard exit, otherwise, unless we are watching the logs, we wouldn't know that they failed to run.

Reply:
There are two points here.

Point 1: running script directly. Agree with @code-asher - we should let users decide which interpreter they want via the shebang.

Point 2: catch scripts exits. From a user standpoint, I imagine I would expect code-server to not start if my entrypoint script had an issue so I think I vote for the hard exit.